docs/operating-scylla/nodetool-commands/snapshot.rst

```diff
@@ -31,8 +31,6 @@ Parameter                                                             Descriptio
 --------------------------------------------------------------------  -------------------------------------------------------------------------------------
 -kc <ktlist>, --kc.list <ktlist>                                      The list of Keyspaces to take snapshot
 --------------------------------------------------------------------  -------------------------------------------------------------------------------------
--p <port> / --port <port>                                             Remote jmx agent port number
```

Comment:
I'm not familiar with the nodetool implementation, but isn't this option also present on native nodetool?
Looks like it's defined here:
https://github.com/scylladb/scylladb/blob/master/tools/scylla-nodetool.cc#L3195

And I guess new desciption of the option on the sourcecode can be used here:
_the port of the REST API of the ScyllaDB node_